NeosModLoader/AssemblyLoader.cs

```diff
@@ -17,7 +18,10 @@ internal static class AssemblyLoader
             try
             {
                 assembliesToLoad = Directory.GetFiles(assembliesDirectory, "*.dll");
```

Comment:
```cs
assembliesToLoad = Directory.GetFiles(assembliesDirectory, "*.dll", SearchOption.AllDirectories);
```

would remove the need for us to do our own recursion

Reply:
Didn't know about that option, makes sense.